site/pages/react-native/signer/authenticating-users/authenticating-with-otp.mdx

```diff
@@ -1,61 +1,142 @@
----
 title: Authenticating Users via One-Time Password (OTP)
 description: Learn how to authenticate users via a one-time password (OTP), sent to their email
+
 ---
 
+import Snippet from "../../../../shared/react-native/setup-guides-snippet.mdx";
+
 # Authenticating Users via OTP
 
-This guide assumes you have already followed the [Setup Guide](/react-native/signer/setup-guide) and have a basic understanding of how to use the Signer in your project.
+<Snippet />
 
-:::info
-For example code on how to use the Signer in your project, please checkout our example apps in our github repository.
+Authenticating a user is easy using the `useAuthenticate()` hook from the `@account-kit/react-native` package.
 
-- [Bare React Native Example](https://github.com/alchemyplatform/aa-sdk/tree/main/examples/react-native-bare-example)
-- [Expo Example](https://github.com/alchemyplatform/aa-sdk/tree/main/examples/react-native-expo-example)
+### Send a One-Time Password (OTP) to a User
 
-:::
+To send an OTP to a user's email, use the `authenticate()` function from the `useAuthenticate()` hook with the `type` set to `email` and the `emailMode` set to `otp`.
 
-## Create a Signer Instance
+```tsx twoslash [sign-in-with-otp.tsx]
+import { useAuthenticate } from "@account-kit/react-native";
+import React, { useState } from "react";
+import { Alert, View, Text, TextInput, Button, Pressable } from "react-native";
 
-```ts twoslash
-// [!include ~/shared/react-native/signer.ts]
-```
+function SignInWithOtp() {
+  const { authenticate } = useAuthenticate();
+  const [email, setEmail] = useState("");
 
-## Authenticate a User
+  const handleUserSignInWithOtp = () => {
+    try {
+      authenticate({
+        email,
+        type: "email",
+        emailMode: "otp",
```

Comment:
this is going to be deprecated in the next couple days, just a heads up. @florrdv is working on that right now